examples/README.md

```diff
@@ -123,21 +123,61 @@ In order to run this examples first you will need to run the dependent services:
 
```

Comment:
This requires docker and docker-compose to be installed. These can be installed via homebrew

Reply:
And make sure you have no other instances of RabbitMQ running on your local machine